IdokladSdk.IntegrationTests/Tests/Clients/ReceivedDocumentPayment/ReceivedDocumentPaymentTests.cs

```diff
@@ -91,6 +93,49 @@ public async Task List_WithFilter_PaymentOptionId_ReturnsListAsync()
             Assert.That(deleted, Is.True);
         }
 
+        [Test]
+        public async Task BatchPostAsync_SuccessfullyPosted()
+        {
+            // Arrange
+            var unpaidInvoiceId = await PostReceivedInvoiceAsync();
+
+            // Act
```

Comment:
V Actu by podle mě mělo být jen volání té metody, kterou testuješ. Zbytek v Arrange a to smazání v // Teardown. Takhle mi z toho není jasné, co za metodu se vlastně testuje.

Reply:
opraveno